plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -53,6 +53,17 @@ def extract_rpm_database
         files[pair.first] = pair[1]
       end
     end
+    # make sure that all parent directories of managed rpm directories are considered
+    # managed
+    dirh.dup.each do |d, e|
+      dir, sep, file = d.rpartition("/")
```

Comment:
Useless assignment to variable - `sep`. Use `_` or `_sep` as a variable name to indicate that it won't be used.<br>Useless assignment to variable - `file`. Use `_` or `_file` as a variable name to indicate that it won't be used.